lua-mosquitto.c

```diff
@@ -539,7 +577,7 @@ static int ctx_threaded_set(lua_State *L)
 static int ctx_option(lua_State *L)
 {
 	ctx_t *ctx = ctx_check(L, 1);
-	enum mosq_opt_t option = luaL_checkint(L, 2);
+	enum mosq_opt_t option = luaL_checkinteger(L, 2);
```

Comment:
this looks like an unrelated change? why do you need this?

Reply:
I run into an compile error, because I forgot to set `LUA_COMPAT_APIINTCASTS`.

Change removed